lib/TInitialState.php

```diff
@@ -82,6 +82,11 @@ protected function publishInitialStateForUser(IUser $user, IRootFolder $rootFold
 			$appManager->isEnabledForUser('circles', $user)
 		);
 
+		$this->initialStateService->provideInitialState(
```

Comment:
to avoid "undefined index" errors, please also add to publishInitialStateForGuest with the matching default

Reply:
adjusted